src/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css

```
@@ -1,6 +1,7 @@
 /* Provided by the Sphinx base theme template at build time */
 @import "../basic.css";
-@import url('https://fonts.googleapis.com/css2?family=Inconsolata:wght@600&display=swap');
+/*@import url('https://fonts.googleapis.com/css2?family=Inconsolata:wght@600&display=swap');*/
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@300;400;600;700&display=swap');
```

```suggestion
@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@300;400;600;700&display=swap');
```

If we are commenting statements, I'd just remove them... Not a fan of commented source code.